src/tiles/item-image.ts

```diff
@@ -21,6 +21,8 @@ export class ItemImage extends LitElement {
 
   @state() private isDeemphasize = false;
 
+  @query('.item-image') _itemImageWaveform!: HTMLImageElement;
```

Comment:
Suggestion: use `private` instead of prefixing with an underscore. You don't really need to use the underscore private convention since typescript natively supports `private` properties.